ff2mpv.js

```diff
@@ -17,6 +38,14 @@ browser.contextMenus.onClicked.addListener((info, tab) => {
             url = info.linkUrl || info.srcUrl || info.selectionText || info.frameUrl;
             if (url) ff2mpv(url);
             break;
+
+        case "ff2mpv_cookies":
+            /* These should be mutually exclusive, but,
+               if they aren't, this is a reasonable priority.
+            */
+            url = info.linkUrl || info.srcUrl || info.selectionText || info.frameUrl;
+            if (url) ff2mpv_cookies(url);
```

Comment:
Instead of a separate `ff2mpv_cookies` function, maybe just have `ff2mpv` take an optional `cookies` parameter?

Something like:

```js
case "ff2mpv_cookies":
    url = ...;
    browser.cookies.getAll({url: url}).then((cookies) => {
        ff2mpv(url, cookies);
    });
    break;
```

Reply:
Yep, the latest commit should have fixed this. Now only the preference menu needs to be implemented.